src/Engine/Physics/Shape.hpp

```diff
@@ -67,6 +63,13 @@ namespace Physics {
             const Plane* GetPlaneData() const;
 
         private:
+            /// Get the wrapped Bullet shape.
+            /**
+             * @return The Bullet shape.
+             */
+            btCollisionShape* GetShape();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Class 'Shape' is unsafe, 'Shape::shape' can leak by wrong usage. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunsafeClassCanLeak)